config.md

```diff
@@ -885,6 +885,7 @@ If `false`, then the gRPC server is not started and no gRPC traffic is accepted.
 
 - Not eligible for live reload.
 - Type: `bool`
+- Default: `true`
```

Comment:
I thought we said we didn't want to default this to true if we're providing a default port as this will otherwise be a breaking change?

Reply:
I wasn't in the conversation, but if the intent is to provide an enabled flag with defaults, the enabled flag should default to false.

Reply:
I have switched it back to default false.